### PR TITLE
Feature: Property label on top

### PR DESCRIPTION
**Pull request by nielslyngsoe**
This Feature enables all Properties to have their label displayed on top.
This is configured on the Property level, meaning it's not part of the DataType config.

**Test notes:**
* Check that any(a few) property editors can be used with Label On Top set to true.
* And see that the label goes on top of the property in the Content Editor.
* Check that it scales properly, label should go on top for all on small screens.

---
_This item has been added to our backlog [AB#8131](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/8131)_

### CODE REVIEW

File: src/Umbraco.Core/Migrations/Upgrade/V_8_10_0/AddPropertyTypeLabelOnTopColumn.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿using System.Linq;
+using Umbraco.Core.Persistence.Dtos;
+
+namespace Umbraco.Core.Migrations.Upgrade.V_8_6_0
```
Comment (bergmania) [inline src/Umbraco.Core/Migrations/Upgrade/V_8_10_0/AddPropertyTypeLabelOnTopColumn.cs:4]: Wrong namespace
```suggestion
namespace Umbraco.Core.Migrations.Upgrade.V_8_10_0
```

**Review by bergmania — COMMENTED** [review]
Just found a single thing

**Comment by warrenbuckley** [comment]
JS code reads fine and @bergmania has reviewed the C# bits I help do for Niels.
Off to test & verify it work as intended one last time

**Comment by warrenbuckley** [comment]
All good and is due for 8.10

**Pull request merged**